app/views/topup/search.html

```diff
@@ -41,7 +41,7 @@ <h2 class="govuk-fieldset__heading">
               </h2>
             </legend>
 
-            <div class="govuk-radios govuk-radios">
+            <div class="govuk-radios govuk-radios--small">
```

Comment:
Agree we should try to make compact this section - but I didn't see all that much difference from using the smaller radios (think the padding is increased for them?). Maybe we can mention we would like to explore in beta making it clear the list is below etc

Reply:
Yes the list is still way down the page isn't it. Addressing this properly in beta sounds good.